src/versjon/versjon_tool.py

```diff
@@ -195,3 +200,20 @@ def run(docs_path, no_stable_index, user_templates):
         # Get the HTML for each page
         with open(stable_dir.joinpath('index.html'), 'w') as index_html:
             index_html.write(index_data)
+
+    if not no_index:
+        index_dir = docs_path
+
+        # Page context
+        page_context = {
+            'page_root': '/'
+        }
+
+        index_data = inject_render.render(
+            template_file='index.html', **general_context,
+            **page_context)
+
+        # Get the HTML for each page
+        with open(stable_dir.joinpath('index.html'), 'w') as index_html:
```

Comment:
Should not be stable_dir here right. Should be documentation root or something like that?

Reply:
wooopsy